src/js/jwt/jwt.js

```diff
@@ -212,6 +213,9 @@ function logout(bounce) {
 
     // Redirect to logout
     if (bounce) {
+        cookie.set('justLoggedOut', 'true', {
```

Comment:
we should follow our naming convention with this cookie, something like
`cs_loggedOut` and also scope similar to `cs_jwt`

Reply:
What exactly do you mean by scope? otherwise changed the name

Reply:
Ah nevermind - I was talking about scoping the domain of the cookie to cloud.redhat.com, but js-cookie does that by default.